packages/serialized-state-manager/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@feature-hub/serialized-state-manager",
+  "version": "0.0.0",
+  "description": "A Feature Service to manage handing over state to the client when server-side rendering.",
```

Comment:
See comment above.

Reply:
> Also please add a link to the website sidebar.

I suggest we add the link when there’s more than empty methods that throw when called. 🙃